.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+**Change Description**
+
+<!-- E.g. a bugfix, feature, refactoring, build related change, etc… What is the change and the motivation for it.  Reference issues where possible.-->
+
+**Implications and Backward Compatibility**
```

Comment:
I think we need to break this section into 2, the more likely section is what APIs does this "break" if any.  Right now we consider the HTTP-RPC API as our primary API.  These would largely be the implications and backwards compatibility notes I think this section is meant to address.

the second class which is rare but severe is concensus changes.  This includes changes to the WebAssembly ABI or any of the processing that affects validation of blocks in the chain.  99% of the time, this section would be blank but, if it isn't then it needs to be called out separately and handled.

Reply:
Good idea.  Basically the intent is to ferret out:
1) things that developers should be aware of before they upgrade
2) documentation that needs to be updated to reflect the changes